### DISCUSSION TITLE
Generate Invite links

### DISCUSSION THREAD

**Discussion by gevera** [directus/directus#4762] [Feature Requests]
As of right now, we can only invite users via email. And it doesn't work out of the box of course, since the SMTP server is not set up. Wouldn't be easier just to generate a unique one time link and as an admin could share the link via messengers or manually via email to the desired user to be? Once the user registers the link is expired. Or if we want to create multiple users of the same role we share a link that doesn't expire right away. Just an idea

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
